doc/classes/MainLoop.xml

```diff
@@ -76,18 +76,19 @@
 			<return type="bool" />
 			<param index="0" name="delta" type="float" />
 			<description>
-				Called each physics frame with the time since the last physics frame as argument ([param delta], in seconds). Equivalent to [method Node._physics_process].
+				Called each physics frame. [param delta] is the time between physics frames in seconds and is determined by [member Engine.physics_ticks_per_second]. Equivalent to [method Node._physics_process].
```

Comment:
I would say "is the logical time between physics ticks".

There are two things there:
* Firstly the physics delta pass is *not* real time, it is logical time. If you were to measure the wall clock time at each physics tick, it would bear no resemblence to the delta here.
* It's good practice to move from referring to physics ticks from frames to calling them ticks, in order to prevent confusion. Physics ticks are not frames. We did agree to standardize this a bit more for 4.x (there's a lot of things that still refer to physics frames in 3.x).